src/index.html

```diff
@@ -1,13 +1,69 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="
+            width=device-width,
+            user-scalable=no,
+            initial-scale=1.0,
+            maximum-scale=1.0,
+            minimum-scale=1.0"
+    >
+
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
+    <title>Product cards</title>
+  </head>
+  <body class="page-content">
+    <div class="product-card" data-qa="card">
+      <img
+        class="product-card__image"
+        src="images/imac.jpeg"
+        alt="imac"
+      >
+
+      <div class="product-card__name">
+        <h1 class="product-card__device-model">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+
+        <h2 class="product-card__code">
+          Product code: 195434
+        </h2>
+      </div>
+
+      <div class="product-card__rating">
+        <div class="product-card__stars stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <h2 class="product-card__reviews">
+          Reviews: 5
+        </h2>
+      </div>
+
+      <div class="product-card__price-section">
+        <h2 class="product-card__price-label">
+          Price:
+        </h2>
+        <h1 class="product-card__price">
+          $2,199
+        </h1>
+      </div>
+
+      <div class="product-card__buy">
+        <a class="product-card__buy-link" data-qa="hover">
+          BUY
+        </a>
+      </div>
```

Comment:
I don't think you really deed this container, just set right styles to the link 🙂